optaplanner-core/src/main/java/org/optaplanner/core/config/solver/DomainAccessType.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.optaplanner.core.config.solver;
+
+public enum DomainAccessType {
```

Comment:
Naming: Do we access a domain? Or do we access a Member of the domain?

Proposal A) DomainAccessType
Proposal B) MemberAccessType

I'd argue that B) is more appropriate

Reply:
This enum might also be used for the SolutionCloner (in which case DomainAccessType is probably more apt)

Reply:
Agreed, my bad.